austrakka/components/sequence/funcs.py

```diff
@@ -451,41 +466,43 @@ def list_sequences(
     )
 
 
-@logger_wraps()
-def purge_sequence(sample_id: str, seq_type: str, skip: bool, force: bool, delete_all: bool):
-    custom_headers = {}
-    _set_mode_header(custom_headers, force, skip)
-    api_path = "/".join([SEQUENCE_PATH, SEQUENCE_BY_SAMPLE_PATH, sample_id, seq_type])
-
-    if delete_all:
-        api_path = api_path + '?deleteAll=true'
-
-    api_delete(
-        path=api_path,
-        custom_headers=custom_headers
-    )
-
-def _build_headers(seq_type, csv_row, force, skip):
+def _build_headers(seq_type, csv_row, force, skip, seq_files: List[SeqFile]):
     # seq type
     headers = {
         SEQTYPE_HEADER: seq_type.value,
     }
     # Put all values from CSV row in headers (Seq_ID and file paths)
     for column in csv_row.index:
-        if column==SEQ_ID_CSV:
+        if column == SEQ_ID_CSV:
             headers[SEQ_ID_HEADER] = csv_row[column]
         else:
             headers[HEADER_MAP[column]] = os.path.basename(csv_row[column])
     _set_mode_header(headers, force, skip)
+    _set_hash_headers(headers, seq_files)
     return headers
 
+
+def _set_hash_headers(headers, seq_files):
+    if len(seq_files) == 2:
+        for seq_file in seq_files:
+            if '_r1' in seq_file.filename:
```

Comment:
This isn't safe; we can't guarantee the files will be named this way. My own manual test failed because the "r1" was capitalised, but in general we can't assume the filename at all. We should go off the header in the files csv, which specifies whether this is file 1 or 2